packages/react-app/src/lib/metamask.js

```diff
@@ -11,3 +15,22 @@ export const addTokenToMetamask = async token => {
     },
   });
 };
+
+export const addChainToMetaMask = async ethereumChain => {
+  const { chainId } = ethereumChain;
+  return window.ethereum.request({
+    method: 'wallet_addEthereumChain',
+    params: [
+      {
+        chainId: utils.hexValue(chainId),
+        chainName: getNetworkName(chainId),
+        nativeCurrency: {
+          name: 'SOME',
```

Comment:
We can have a map in our `lib/constants` file to store symbol and names of native currencies for the corresponding chainId